app/Jobs/ProcessSubmission.php

```diff
@@ -150,7 +150,7 @@ public function handle()
      */
     public function failed(\Throwable $exception) : void
     {
-        \Log::warning("Failed to process {$this->filename}");
+        \Log::warning("Failed to process {$this->filename} with message {$exception->getMessage()}");
```

Comment:
The message appears without issue for malformed XML. 
Super nitpicky, IMO for readability, can we add a `:` to separate the message from our text? It would turn

`[2023-09-06 18:08:14] testing.WARNING: Failed to process TestCompressionExample_-__-_22b78c02-7901-4973-bd4f-55db009b88a4_-_.xml with message Attempt to read property "testStatus" on null` 

to 

`[2023-09-06 18:08:14] testing.WARNING: Failed to process TestCompressionExample_-__-_22b78c02-7901-4973-bd4f-55db009b88a4_-_.xml with message: Attempt to read property "testStatus" on null`

Feel free to ignore this if you don't think it's needed

Reply:
Good catch!  Adding a colon definitely helps it be more clear about where the actual message starts.